packages/react-components/components/SimpleMessagingCard.tsx

```diff
@@ -20,11 +20,22 @@ export default function SimpleMessagingCard({
   callToActions,
   testID,
 }: Props) {
-  const icon = React.isValidElement(iconProp) ? (
-    iconProp
+  const icon = iconProp ? (
+    React.isValidElement(iconProp) ? (
+      iconProp
+    ) : (
+      <Image
+        // @ts-ignore isValidElement check above ensures image is an image source type
+        source={iconProp}
+        resizeMode="contain"
+        style={
+          Object.prototype.hasOwnProperty.call(iconProp, 'uri') ? styles.remoteIcon : undefined
+        }
+        testID={`${testID}/Icon`}
+      />
+    )
   ) : (
-    // @ts-ignore isValidElement check above ensures image is an image source type
-    <Image source={iconProp} resizeMode="contain" />
+    undefined
```

Comment:
Worthwhile to have a default icon for when one isn't provided?

Reply:
Yes I thought about it then decided not to do it as this gives us flexbility to intentionally not display an icon.